packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/ios/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,523 @@
+	// The fields for the one-time code.
+    @State var codeData: AuthenticatorCode? = nil
+
+    @State var qrCode: UIImage? = nil
+	// The entered code.
+    @State var code: String = ""
+
+    var body: some View {
+        Form {
+            if let qrCode = enrollData?.qrCode {
+                Section("Enrollment Code") {
+                    if let name = enrollData!.name {
+                        Text(name)
+                    }
+                    Image(uiImage: qrCode)
+                        .resizable()
+                        .scaledToFit()
+                        .frame(width: 200, height: 200)
+                    Text("Enroll by scanning this code in the app or use the following code:")
+                    Text((enrollData?.sharedSecret)!)
+                }
+            }
+            // Display only if there's a message.
+            if let message = message {
+                Section("Message") {
+                    HStack {
+                        Image(systemName: "exclamationmark.triangle.fill")
+                            .foregroundColor(Color.yellow)
+                        Text(message)
+                    }
+                }
+            }
+            Section("Code") {
+                let textLabel = (codeData?.entryField.label != nil) ?
+                codeData?.entryField.label : "Enter the code"
+                TextField(textLabel!,text: $code)
+                    .textInputAutocapitalization(.never)
+                    .disableAutocorrection(true)
+                    .keyboardType(.numberPad)
+            }
+            Section() {
+                Button("Continue") {
+                    continueSignin()
+                }
+                Button("Cancel Sign In", role: .destructive) {
+                    showingSignIn = false
+                }
+            }
+
+        }
+        .onAppear() {
+            if let signInController = SignInController.shared {
+                enrollData = signInController.authenticatorEnrollment()
+                codeData = signInController.authenticatorCodeFields()
+            }
+        }
+    }
+
+    func skip() {
+        print("SKIP")
+
+    }
+
+    func continueSignin() {
+        print("Continue")
+        SignInController.shared?.authenticateWithCode(code: code) {result in
+             isWaiting = false
+            switch result {
+            case .success(let result):
+                isWaiting = false
+                    // Succesful sign in
+                    signedIn = true
+                    showingSignIn = false
+                } else if let message = result.message {
+                    // Show a message on this form.
+                    self.message = message.message
+                } else {
+                    currentState = SignInController.shared!.currentRemediation!.type
+                }
+
+            case .failure(let error):
+                // Handle the error
+            }
+        }
+    }
+}
+```
+
```

Comment:
Again, I'd love to see a screenshot of this here.

Reply:
What screenshot (or shots): The enroll authenticator form before it's filled out? The same but with fake values? Something else? What about the screen for choosing an authenticator?

Reply:
The code immediately above is for the form displaying the QR Code, secret key, and spaces for entering the one-time code, so that's what I would like to see. Possibly that could be a mock up as we've discussed previously with Rob, but to save time jsut a straight screenshot of the form would do. Doesn't need fake values added into the form. But something matching the screenshot I've got in mine.

Reply:
Added text and an image. It will show up in the next commit.